src/main/java/io/cbitler/stealingartefacts/StealingArtefactsPlugin.java

```diff
@@ -325,4 +339,28 @@ public boolean isInPisc(WorldPoint position) {
 
         return false;
     }
+
+    /**
+     * Check if the applicable guards is facing the lured direction
+     * @param guard The NPC object to check
+     * @return True if guard is facing correct position, otherwise false
+     */
+    public boolean isGuardLured(NPC guard) {
+        boolean isLured = false;
+        if (guard.getWorldLocation().distanceTo(EAST_GUARD_POS) == 0 && guard.getCurrentOrientation() == Constants.SOUTH) {
```

Comment:
Nitpick: We should probably move each of the distanceTo checks into variables so that we aren't repeating the same logic twice (this will make it easier to maintain)